src/ui/main.qml

```diff
@@ -348,6 +348,82 @@ Window {
         }
     }
 
+    Connections {
+        target: VPNErrorHandler
+        function onSubscriptionGeneric() {
+            mainStackView.push("qrc:/ui/views/ViewErrorFullScreen.qml", {
+                // Problem confirming subscription...
+                headlineText: VPNl18n.ProblemConfirmingSubscription,
+
+                // An unexpected error has occured, please try again.
+                // Visit our help center below to learn more about how to troubleshoot our VPN.
+                errorMessage: VPNl18n.RestoreUnexpectedError,
+
+                // Try again
+                buttonText: VPNl18n.SubscriptionTryAgain,
+                buttonObjectName: "errorTryAgainButton",
+                buttonOnClick: mainStackView.pop,
+                signOffLinkVisible: false,
+                getHelpLinkVisible: true
+            });
+        }
+
+        function onNoSubscriptionFound() {
+            mainStackView.push("qrc:/ui/views/ViewErrorFullScreen.qml", {
+                // Problem confirming subscription...
+                headlineText: VPNl18n.ProblemConfirmingSubscription,
+
+                // Sorry we are unable to find your subscription, please try again.
+                // Visit our help center below to learn more about how to troubleshoot our VPN.
+                errorMessage: VPNl18n.NoSubscriptionFoundError,
```

Comment:
If this text in this comment is final we'll need to add this as a new string in strings.yaml.